docker-compose/grafana/provisioning/dashboards/nats-surveyor-dashboard.json

```
@@ -776,6 +776,900 @@
         "x": 0,
         "y": 4
       },
+      "id": 248,
+      "panels": [],
+      "title": "Jetstream Global Status",
```

Fine to change to this?

```suggestion
      "title": "JetStream Status",
```

---

Yes, makes sense.. updated!